lib/rules/value-no-vendor-prefix/index.mjs

```diff
@@ -52,8 +53,23 @@ const rule = (primary, secondaryOptions, context) => {
 				return;
 			}
 
-			if (optionsMatches(secondaryOptions, 'ignoreValues', vendor.unprefixed(value))) {
-				return;
+			if (secondaryOptions?.ignoreValues) {
+				const isIgnored = secondaryOptions.ignoreValues.some(
+					(/** @type { string | RegExp } */ element) => {
```

Comment:
[suggest] It's possible to remove the argument `@type`:

```suggestion
					(element) => {
```

by specifying type parameters to `rule`:

```diff
-/** @type {import('stylelint').Rule} */
+/** @type {import('stylelint').Rule<boolean, Array<string | RegExp>>} */
 const rule = (primary, secondaryOptions, context) => {
```